src/runtime/auto-prompt-manager.js

```diff
@@ -298,13 +298,33 @@ export class AutoPromptManager {
           return false;
         }
 
-        // If the user has reached maxImpressionsPerWeek, don't show the prompt.
+        // If the user has reached the maxImpressions, and
+        // maxImpressionsResultingHideSeconds has not yet passed, don't show the
+        // prompt.
+        if (
+          autoPromptConfig.impressionConfig.maxImpressions !== undefined &&
```

Comment:
You should just be able to do `if (autoPromptConfig.impressionConfig.maxImpressions && ..)`

Reply:
Oh yeah, absolutely. Done.